frontend/src/containers/SignUp.js

```diff
@@ -73,6 +80,7 @@ class SignUp extends Component {
               <p>{`To create your Account simply fill the short form below`}</p>
               <hr />
               <form onSubmit={e => this.onSignUpHandler(e)}>
+                {error ? <div className="notification is-danger">{error}</div> : null}
```

Comment:
It could be rewritten to conditional rendering form `error && <div />`. Also, having a default value for error state makes the code easier to read, so adding that to the `state = { error: null }` will be helpful